src/cart/src/gurt/dlog.c

```diff
@@ -641,8 +641,18 @@ d_log_open(char *tag, int maxfac_hint, int default_mask, int stderr_mask,
 			fprintf(stderr, "strdup failed.\n");
 			goto error;
 		}
-		mst.logfd =
-		    open(mst.logfile, log_flags, 0666);
+		/* merge stderr into log file, to aggregate and order with
+		 * messages from Mercury/libfabric
+		 */
+		if (freopen(mst.logfile, truncate ? "w" : "a",
+			    stderr) == NULL) {
+			fprintf(stderr, "d_log_open: cannot open %s: %s\n",
+				mst.logfile, strerror(errno));
+			goto error;
+		}
```

Comment:
Would be great to log to both stderr AND the log file if possible.

Reply:
Hmmm, it looks a bit tricky to achieve...